nipype/interfaces/base.py

```diff
@@ -1761,10 +1781,7 @@ class StdOutCommandLine(CommandLine):
     input_spec = StdOutCommandLineInputSpec
 
     def _gen_filename(self, name):
-        if name == 'out_file':
-            return self._gen_outfilename()
-        else:
-            return None
+        return self._gen_outfilename() if name == 'out_file' else None
```

Comment:
Functions return `None` implicitly. This could just drop the `else` condition.

```Python
def _getn_filename(self, name):
    if name == 'out_file':
        return self._gen_outfilename()
```

I'm not sure which one is easier to understand. Your choice.

Reply:
I generally prefer explicit over implicit ...